### DISCUSSION TITLE
Error Load images

### DISCUSSION THREAD

**Discussion by hugopvl** [directus/directus#8663] [Archive]
Rc.96
Hi guys, is the same happening to someone, doesn't load all the images of the items?

![image](https://user-images.githubusercontent.com/81326296/136593968-525a634c-d420-40f9-be45-5ad7f0e2da37.png)

**Comment by rijkvanzanten** [comment]
For potential bugs, Issues is a better place 🙂 Hard to tell without more detail on what the exact setup is, and any potential error messages

**Reply by hugopvl** [reply]
yes, you're absolutely right, i'll put it there, sorry
